src/contracts/Event/RenderGroupEvent.php

```diff
@@ -60,4 +60,12 @@ public function getParameters(): array
     {
         return $this->parameters;
     }
+
+    /**
+     * @param array<mixed> $parameters
+     */
+    public function setParameters(array $parameters): void
```

Comment:
Consider creating `addParameter()` / `addParameters()` methods. With these, there would be no need to acquire already present parameters, merge them, and set - which is error prone.

Instead a single method call would be used.